src/Perspex.Controls/ControlValidationStatus.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Perspex.Data;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Perspex.Controls
+{
+    public class ControlValidationStatus : ValidationStatus, INotifyPropertyChanged
```

Comment:
Hmm, this isn't how I imagined validation status to be... I had imagined that each property would have a validation status. I could imagine a control which had more than one property that be viewmodel-validated for example.

At the moment the validation statuses seem to be indexed by the type of plugin reporting the validation error, is that correct? I can't really see a use for that on the other hand.